ui-modules/blueprint-composer/app/views/main/graphical/edit/add/add.js

```diff
@@ -51,8 +52,11 @@ export function GraphicalEditAddController($scope, $filter, $state, $stateParams
             break;
     }
 
-    $scope.catalogItemsPerPage = 24;
-
+    if (!$scope.familiesToShow) $scope.familiesToShow = [ $scope.family ];
```

Comment:
Could you use curly brackets for the `if` statement please? That's to be consistent with the rest of the codebase and also make it easier to read

Reply:
I cannot find any reference if this on templates